### DISCUSSION TITLE
kyu & honor reset, still unsolved, help?

### DISCUSSION THREAD

**Discussion by victormarcos1** [codewars/codewars.com#3074] [Q&A]
So I'm a 3 kyu with 1100+ honor, and then my stats reset to 6 kyu with 80+ honor, what could've possibly happened?

at the time before it happened, I was solving a 2 kyu Python problem, and when I had earned my 200 points from solving it (900 to 1100), it just dropped down.

**Comment by Blind4Basics** [comment]
> Just to clarify, no I was NOT cheating

Ofc you were, come on...

**Reply by victormarcos1** [reply]
I understand your concern, but I assure you that I take my work on Codewars seriously and do not engage in any form of cheating. It's worth noting that none of the questions I answered on Codewars were flagged or reported as suspicious. If my points were deducted, it was genuinely unexpected on my part, and I posted the question to seek clarification and resolve the issue transparently. I believe in open communication and was hoping for assistance or guidance on what might have happened.

Rather than making unverified accusations, I'd appreciate it if we could discuss this matter calmly and professionally. If you have any evidence or information related to the points deduction, please share it, and I'd be happy to address it. Alternatively, if you know of any resources or support that could help us resolve this, I'm open to your suggestions.

**Reply by Blind4Basics** [reply]
>  none of the questions I answered on Codewars were flagged or reported as suspicious.

Ofc they were. You just don't know about it.
My "accusations" are backed up by evidences. Which, for obvious reasons, won't be discussed openly. 

Considering your account it will stay locked. Your options are the following:
1. keep it that way
2. delete it and start a new account. And this time, don't cheat (also applies to point 1, ofc). I'd personally suggest you go that way.

Just in case there is a misunderstanding about "cheating": 
- grinding solutions on the web/using code you didn't create yourself is considered cheating
- adding comments or manual tests, or console.log statements to an existing solution _doesn't_ make you author of that solution.
